db/model/sample.js

```diff
@@ -126,6 +126,8 @@ module.exports = function sample(seq, dataTypes) {
                 'infoRange',
                 'okRange',
                 'valueLabel',
+                "valueType",
```

Comment:
we use single quotes

(`npm run jscs` will check for code style--you can also set up a sublime editor plugin to show you violations in realtime in your editor, if you use sublime)